VideoApi/VideoApi.IntegrationTests/Steps/CommonSteps.cs

```diff
@@ -142,97 +142,35 @@ public async Task GivenIHaveManyConferences()
             });
         }
 
-        [Given(@"I have several conferences with users")]
+        [Given(@"I have several conferences with hearing venue name")]
         public async Task GivenIHaveSeveralConferencesWithUsers()
```

Comment:
I know this step isn't used for any other tests but the VHO venue list one but can you rename this common step so it isn't accidently reused. Maybe something like GivenIHaveSeveralConferencesAtDifferentVenues.